source/about/faq.rst

```diff
@@ -2,89 +2,56 @@
-    * Ore
+Sponge is a new versatile Minecraft API. It was made to enrich your Minecraft experience by allowing plugins to add
+functionality to Minecraft. Read more about Sponge here: :doc:`introduction`.
 
-Further information regarding the different parts of Sponge i.e. how to contribute can be found here: :doc:`/contributing/index`
+What is required to run Sponge?
+-------------------------------
 
-Is There an Official Plugin Repository?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Sponge (and Minecraft) needs the Java Runtime Environment to run properly. You will obviously need a computer to run
+the server on too, besides that nothing is required. Learn more about choosing the correct Java version here:
+:doc:`../server/getting-started/jre/`
 
-Yes, and it is called Ore. Development has already started on GitHub.
+Where do i get Sponge?
+----------------------
 
-What Version of Java Is Sponge Using?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Sponge currently isn't finished but there are development builds of the SpongeCoremod available: :doc:`/downloads`
+Unfortunately there currently aren't any build of SpongeVanilla available.
 
-Sponge will be using Java 1.6, which is the minimum version supported by vanilla Minecraft and Forge.
+Which different versions exist? / Whats the difference between the versions?
+----------------------------------------------------------------------------
 
-I Haven't Kept Up. What Happened to Bukkit?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Sponge Project currently develops two implementations which both use the SpongeAPI:
 
-The Bukkit project halted further development of their API and server modification. Shortly thereafter, one of the
-contributors to Bukkit sent a DMCA takedown notice to stop further distribution of CraftBukkit. He was within his legal
-right. Downloads, as well as source code, for CraftBukkit and its derivatives (such as Spigot and Cauldron) are no
-longer publicly available.
+(1) **Sponge**, a coremod for Minecraft Forge, which is an existing Minecraft modding framework famous for spurring
+    the Minecraft modding scene. Forge lacks a cross-version API, and this is where Sponge steps in. Sponge allows
+    server owners to deploy Sponge plugins with ease, making server management easier.
 
-What Happened to the Granite Implementation?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(2) **SpongeVanilla**, a stand-alone implementation of the Sponge API, running on top of the vanilla Minecraft server.
+    (SpongeVanilla was formerly known as Granite, until the development teams merged).
 
-The developers of Granite have joined the Sponge project, and Granite was renamed SpongeVanilla. It is now an officially
-supported Sponge API implementation, developed in parallel with Sponge coremod.
+Where do i get Plugins for Sponge?
+----------------------------------
 
+Yes, and it is called Ore. Development has already started on GitHub. Temporarily plugins are hosted on the
```

Comment:
"Where do i get Plugins for Sponge?"
i -> I

And the response should not be "Yes", for a where question.
Temporarily should have a comma after it, to signify a pause.